src/util.rs

```diff
@@ -92,3 +96,24 @@ mod tests {
         assert_eq!(100, limit(1000, 10, 100));
     }
 }
+
+pub fn start_daemon<S,I>(program: &str,args: &Vec<String>) -> io::Result<std::process::Child>{
+    #[cfg(not(windows))]
+    let spawned = Command::new(program)
+        .args(args)
+        .before_exec(|| {
+            // Detach forked process from Alacritty. This will cause
+            // init or whatever to clean up child processes for us.
+            unsafe { ::libc::daemon(1, 0); }
+            Ok(())
+        })
+        .spawn();
+
+    #[cfg(windows)]
+    let spawned = Command::new(program)
+    .args(args)
+    .spawn();
+
+    return spawned;
+
+}
```

Comment:
Since 100% of the method body is split between Windows and macOS/Linux/BSD, two separate methods should be defined and annotated.

It should look something like this instead:
```rust
#[cfg(not(windows))]
pub fn start_daemon(...) -> ... {
    ...
}

#[cfg(windows)]
pub fn start_daemon(...) -> ... {
    ...
}
```

Instead of using `return spawned;`, which is unidiomatic Rust code, you should also just remove the semicolon after `.spawn();` and delete the `return ...;` statement completely.

Please also make sure there's a newline at the end of the file. This is currently missing.